### DISCUSSION TITLE
Model visualized in AR is in wrong orientation

### DISCUSSION THREAD

**Discussion by NitinJ88** [google/model-viewer#4612] [Q&A]
### Description

Model visualized in AR is in wrong orientation. XY coordinate is not correct. see the attached images. Is there any way to correct it?

#### Live Demo

https://glitch.com/edit/#!/model-viewer

![AR_visual_oriantation_XY](https://github.com/google/model-viewer/assets/86343802/71d275b8-6259-4d05-8484-50577a7018c8)

![AR_visual_oriantation_XZ](https://github.com/google/model-viewer/assets/86343802/7c443fff-7973-4235-bf9c-56b292c1ca70)

### Version

- model-viewer: v3.2.1

### Browser Affected

- [*] Chrome, version: 120.0.6099.130
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [*] Android
- [* ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR

- [*] WebXR
- [ ] SceneViewer
- [*] QuickLook

**Comment by elalish** [comment]
Yes, glTF defines Y as up for all models, so a proper glTF/GLB exporter should make that change for you if your modeling software uses a different coordinate system like yours does. If it's hard to fix the export, you can use our [`orientation`](https://modelviewer.dev/docs/#entrydocs-scenegraph-attributes-orientation) attribute to fix it in MV.
